packages/dd-trace/src/debugger/devtools_client/send.js

```diff
@@ -37,5 +39,13 @@ function send (message, logger, snapshot, cb) {
     'debugger.snapshot': snapshot
   }
 
-  request(JSON.stringify(payload), opts, cb)
+  let json = JSON.stringify(payload)
+
+  if (Buffer.byteLength(json) > MAX_PAYLOAD_SIZE) {
+    // TODO: This is a very crude way to handle large payloads. Proper pruning will be implemented later (DEBUG-2624)
+    delete payload['debugger.snapshot']
+    json = JSON.stringify(payload)
+  }
```

Comment:
(nitpick) is this any thing different from this? 

```suggestion
  if (Buffer.byteLength(json) > MAX_PAYLOAD_SIZE) {
    // TODO: This is a very crude way to handle large payloads. Proper pruning will be implemented later (DEBUG-2624)
    delete payload['debugger.snapshot']
  }
  const json = JSON.stringify(payload)
```
it seems unnecessary to potentially `JSON.stringify` twice. Let me know if this makes sense 😄